### PR TITLE
Add bulk_download to mg_toolkit suite

### PR DESCRIPTION
**Pull request by RZ9082**
FOR CONTRIBUTOR:
* [x] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] License permits unrestricted use (educational + commercial)
* [x] This PR adds a new tool or tool collection
* [x] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

### CODE REVIEW

File: tools/mg_toolkit/bulk_download.xml [hunk 1]
```diff
@@ -0,0 +1,870 @@
...
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="creators" />
+    <expand macro="biotools" />
+    <expand macro="requirements" />
+    <command><![CDATA[
+        mg-toolkit -d bulk_download -a $accession
+        -p $version_result_group.pipeline
+        -g $version_result_group.result_group.result_group &&
+        if [ -f ${accession}/${accession}_metadata.tsv ] ; then mv ${accession}/${accession}_metadata.tsv $metadata; fi &&
```
Comment (bgruening) [inline tools/mg_toolkit/bulk_download.xml:13]:
```suggestion
        mv ${accession}/${accession}_metadata.tsv '$metadata' > /dev/null || true &&
```

Could be an alternative.

File: tools/mg_toolkit/.shed.yml [hunk 2]
```diff
@@ -5,5 +5,15 @@ homepage_url: https://github.com/EBI-Metagenomics/emg-toolkit
...
 categories:
 - Metagenomics
+auto_tool_repositories:
+  name_template: "{{ tool_id }}"
+  description_template: "Wrapper for the mg-toolkit: {{ tool_name }}"
+suite:
+  name: "suite_mg_toolkit"
+  description: "Metagenomic toolkit for downloading metadata and study results"
+  long_description: |
+    Metagenomics toolkit enables scientists to download all of the sample metadata for a given study or sequence to a single csv file,
```
Comment (bgruening) [inline tools/mg_toolkit/.shed.yml:18]: tsv?

File: tools/mg_toolkit/bulk_download.xml [hunk 3]
```diff
@@ -0,0 +1,870 @@
...
+                    </assert_contents>
+                </element>
+            </output_collection>
+        </test>
+    </tests>
+    <help>
+<![CDATA[
+Download result files in bulks for an entire study from the MGnify archive.
+
+options:
```
Comment (bgruening) [inline tools/mg_toolkit/bulk_download.xml:838]: This should be removed or enhanced.

File: tools/mg_toolkit/bulk_download.xml [hunk 4]
```diff
@@ -0,0 +1,870 @@
...
+                         - taxonomic_analysis_unite (>= 5.0)
+                         - taxonomic_analysis_motu  (>= 5.0)
+                         - pathways_and_systems (>= 5.0)
+
+License
+-------
+* `Apache-2.0 license <https://raw.githubusercontent.com/EBI-Metagenomics/emg-toolkit/master/LICENSE>`_
+
+]]>
+    </help>
```
Comment (bgruening) [inline tools/mg_toolkit/bulk_download.xml:869]: citation?

File: tools/mg_toolkit/.shed.yml (hunk 2 above)
Comment (RZ9082) [reply tools/mg_toolkit/.shed.yml:19]: Actually csv, this refers to the first tool in the toolkit

**Review by bgruening — APPROVED** [review]

**Pull request merged**